qiskit/wrapper/configrc/_settings.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Module that holds the runtime location of qiskitrc file.
+"""
+
+QISKIT_RC_FILE = None
```

Comment:
Can this variable contain a value other than `~/.qiskit/qiskitrc`? In the usual spirit, if we can avoid having a global variable, other implementation would be preferred.

Reply:
It can be `None`.  It is used not only to point to the `qiskitrc` file, but to tell the routines if it can not be found. It is set from `None` to the location of the `qiskitrc` file at runtime if it is there.